src/main/scala/gitbucket/core/service/PullRequestService.scala

```diff
@@ -293,6 +302,66 @@ trait PullRequestService {
     }
   }
 
+  def updatePullRequestsByApi(
+    repository: RepositoryInfo,
+    issueId: Int,
+    loginAccount: Account,
+    settings: SystemSettings,
+    title: Option[String],
+    body: Option[String],
+    state: Option[String],
+    base: Option[String]
+  )(
+    implicit s: Session,
+    c: JsonFormat.Context
+  ): Unit = {
+    getPullRequest(repository.owner, repository.name, issueId) match {
```

Comment:
In this case, using `foreach` instead of `match` would be better so that we can get rid of `case _ =>`.

Reply:
@takezoe Thanks for your review! I fixed PR!
BTW I think we should add function to let message when repository updated by api. Currently api can change (PR, Issue, Issue comment, etc. ) silently... that causes lack of transparency.

Reply:
> BTW I think we should add function to let message when repository updated by api.

A kind of auditing? Do you have any idea where or how such messages will be shown?

Reply:
When user change tag, we can see history on issue like below.

![image](https://user-images.githubusercontent.com/26293997/94548287-744f5d00-028b-11eb-9dd7-98a523745baf.png)

IMHO It would be more transparent if this kind of history show up even when some thing got changed by api.
But it is another issue.

Reply:
I noticed that I just can use [createComment](https://github.com/gitbucket/gitbucket/blob/8dbcbb54851feffca390957990baffc43697b08d/src/main/scala/gitbucket/core/service/IssuesService.scala#L516) to leave history.
So I updated my PR to leave comment. https://github.com/gitbucket/gitbucket/pull/2557/commits/b7506459a5b3c02d0c0d84f460787996c581b582

Reply:
Great thanks!